### ISSUE TITLE
Tests currently fail on Windows

### ISSUE THREAD

**Issue by rneatherway** [fsprojects/projekt#24]
Might be something to do with using fsunit, which sometimes causes these kinds of problems. Seem to have NuGet 2.6.4 locally and something is expecting 2.6.3.

```
1) Test Error : Projekt.Tests.addFile should fail if file in project
   System.IO.FileLoadException : Could not load file or assembly 'nunit.framework, Version=2.6.
3.13283, Culture=neutral, PublicKeyToken=96d09a1eb7f44a77' or one of its dependencies. The loca
ted assembly's manifest definition does not match the assembly reference. (Exception from HRESU
LT: 0x80131040)
   at Projekt.Tests.addFile should fail if file in project()
```

**Comment by kjnilsson** [comment]
why does a testing framework need to be strongly named? just don't get it.
gah. :)

**Comment by rneatherway** [comment]
Agreed... tiresome. It also works on fine on Mono, my favourite kind of issue, not.

**Comment by rneatherway** [comment]
Fixed by #31
